Makefile

```diff
@@ -0,0 +1,40 @@
+NAME ?= "KialiTestService"
+BINARY_NAME ?=kiali-test-service
+# Identifies the current build.
+# These will be embedded in the app and displayed when it starts.
+VERSION ?= 0.0.1.Final-SNAPSHOT
+COMMIT_HASH ?= $(shell git rev-parse HEAD)
+
+# Environment variables set when running the Go compiler.
+GO_BUILD_ENVVARS = \
+        GOOS=linux \
+        GOARCH=amd64 \
+        CGO_ENABLED=0 \
+
+DOCKER_NAME ?= kiali/kiali-test-service
+DOCKER_TAG ?= latest
+
+all: build
+
+build:
+	@echo Building...
+	${GO_BUILD_ENVVARS} go build \
+		-o ${GOPATH}/bin/${BINARY_NAME} -ldflags "-X main.name=${NAME} -X main.version=${VERSION} -X main.commitHash=${COMMIT_HASH}"
+
+docker-build:
+	@echo Building the docker image : ${DOCKER_NAME}:${DOCKER_TAG}
+	@mkdir -p _output/docker
+	@cp ${GOPATH}/bin/kiali-test-service _output/docker
+	@cp deploy/docker/* _output/docker
+	@docker build -t ${DOCKER_NAME}:${DOCKER_TAG} _output/docker
+
+deploy-openshift:
```

Comment:
do we want to keep the make target names consistent with the other kiali make file? We use "openshift-deploy" in swscore.

Reply:
I was thinking we would have 'deploy-openshift' and 'deploy-kubernetes', and then I could use 'make deploy' and tab to autocomplete the options.

But this is a minor opinion

Reply:
that's a good idea - feel free to submit a PR for swscore to change the make targets over there for consistency. swscore makefile does already have k8s-deploy and openshift-deploy targets that can be renamed

Reply:
Actually, I take this back. I may want instead to type 'make openshift' and see all the OpenShift options I have (and there might not be a 1:1 mapping between what options we will need for OpenShift and what we will need for K8S or another platform).

This will also make it more inline with swscore